src/components/button-toggle/button-toggle.spec.js

```diff
@@ -37,6 +38,73 @@ describe('ButtonToggle', () => {
     });
   });
 
+
+  describe('HiddenCheckableInput', () => {
+    let propOnBlur;
+    let groupContextOnBlur;
+
+    let groupContextOnFocus;
+
+    let groupContextOnMouseEnter;
+
+    let groupContextOnMouseLeave;
+
+    let wrapper;
+
+    beforeEach(() => {
+      propOnBlur = jest.fn();
+      groupContextOnBlur = jest.fn();
+
+      groupContextOnFocus = jest.fn();
+
+      groupContextOnMouseEnter = jest.fn();
+
+      groupContextOnMouseLeave = jest.fn();
+
+      wrapper = renderWithContext(
+        {
+          onBlur: propOnBlur
+        },
+        {
+          onBlur: groupContextOnBlur,
+          onFocus: groupContextOnFocus,
+          onMouseEnter: groupContextOnMouseEnter,
+          onMouseLeave: groupContextOnMouseLeave
+        },
+      );
+    });
+
+
+    it('triggers onFocus callbacks passed from props and context', () => {
+      wrapper.find(StyledButtonToggleInput).props().onFocus();
+      expect(groupContextOnFocus).toHaveBeenCalled();
+    });
+
+    it('triggers onBlur callbacks passed from props and context', () => {
+      wrapper.find(StyledButtonToggleInput).props().onBlur();
+      expect(propOnBlur).toHaveBeenCalled();
+      expect(groupContextOnBlur).toHaveBeenCalled();
+    });
+
+    it('triggers onMouseEnter callback passed from context', () => {
+      wrapper.find(StyledButtonToggleLabel).props().onMouseEnter();
+      expect(groupContextOnMouseEnter).toHaveBeenCalled();
+    });
+
+    it('triggers onMouseLeave callback passed from context', () => {
+      wrapper.find(StyledButtonToggleLabel).props().onMouseLeave();
+      expect(groupContextOnMouseLeave).toHaveBeenCalled();
+    });
+
+    it('nothing happens if onBlur callbacks are not provided', () => {
```

Comment:
And same here. `it('does nothing if onBlur callbacks are not provided',` seems more appropriate to me.